[hexo] Cannot start local hexo server "unknown block tag: endraw"

Trying to run local hexo server from fresh clone, on Linux Mint:
```
$ git clone https://github.com/meteor/guide.git
$ cd guide
$ git submodule update --recursive --init
$ cd site
$ npm install
$ npm start

> meteor-guide@1.0.0 start /home/chris/meteor-docs/guide/site
> npm run build && chexo meteor-hexo-config -- server


> meteor-guide@1.0.0 build /home/chris/meteor-docs/guide/site
> chexo meteor-hexo-config -- generate

INFO  Config based on 2 files
INFO  Start processing
FATAL Something's wrong. Maybe you can find the solution here: https://hexo.io/docs/troubleshooting.html
Nunjucks Error:  [Line 6, Column 13] unknown block tag: endraw
    =====               Context Dump               =====
    === (line number probably different from source) ===
  1 | <h2 id="breaking-changes">Breaking changes</h2>
  2 | 
  3 | <p>These are all the <em>breaking changes</em> &mdash; that is, changes you absolutely have to worry about if you are updating your app from 1.3.x to 1.4. However, we recommend that you also consider the <em>recommended</em> changes <a href="#recommendations">listed below</a>.</p>
  4 | <h3 id="The-babel-runtime-npm-is-usually-required"><a href="#The-babel-runtime-npm-is-usually-required" class="headerlink" title="The babel-runtime npm is usually required"></a>The <code>babel-runtime</code> npm is usually required</h3><p><a name="babel-runtime-required"></a></p>
  5 | <p>The <code>babel-runtime</code> npm package is generally required as a dependency since the Meteor <code>babel-runtime</code> package no longer attempts to provide custom implementations of Babel helper functions. To install the <code>babel-runtime</code> npm, run the following command in any Meteor application:</p>
  6 | <!--￼0-->{% endraw %}
  7 | 
  8 | 
  9 | 
  10 | <p>One place this is particularly useful is in <a href="https://github.com/meteor/todos/commit/3963a65d96cd7ef235a95d5e3a331d6f0606f70f">test files that are only intended to run on the client or the server</a> &mdash; you can now use <code>import</code> wherever you like, without having to organize your tests in <code>client</code> or <code>server</code> directories.</p>
  11 | 
    =====             Context Dump Ends            =====
    at formatNunjucksError (/home/chris/meteor-docs/guide/site/node_modules/hexo/lib/extend/tag.js:102:13)
    at Promise.fromCallback.catch.err (/home/chris/meteor-docs/guide/site/node_modules/hexo/lib/extend/tag.js:124:34)
    at tryCatcher (/home/chris/meteor-docs/guide/site/node_modules/hexo/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/home/chris/meteor-docs/guide/site/node_modules/hexo/node_modules/bluebird/js/release/promise.js:517:31)
    at Promise._settlePromise (/home/chris/meteor-docs/guide/site/node_modules/hexo/node_modules/bluebird/js/release/promise.js:574:18)
    at Promise._settlePromise0 (/home/chris/meteor-docs/guide/site/node_modules/hexo/node_modules/bluebird/js/release/promise.js:619:10)
    at Promise._settlePromises (/home/chris/meteor-docs/guide/site/node_modules/hexo/node_modules/bluebird/js/release/promise.js:695:18)
    at _drainQueueStep (/home/chris/meteor-docs/guide/site/node_modules/hexo/node_modules/bluebird/js/release/async.js:138:12)
    at _drainQueue (/home/chris/meteor-docs/guide/site/node_modules/hexo/node_modules/bluebird/js/release/async.js:131:9)
    at Async._drainQueues (/home/chris/meteor-docs/guide/site/node_modules/hexo/node_modules/bluebird/js/release/async.js:147:5)
    at Immediate.Async.drainQueues [as _onImmediate] (/home/chris/meteor-docs/guide/site/node_modules/hexo/node_modules/bluebird/js/release/async.js:17:14)
    at runCallback (timers.js:696:18)
    at tryOnImmediate (timers.js:667:5)
    at processImmediate (timers.js:649:5)
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! meteor-guide@1.0.0 build: `chexo meteor-hexo-config -- generate`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the meteor-guide@1.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

--- a/content/1.3-migration.md
+++ b/content/1.3-migration.md
@@ -9,9 +9,9 @@ discourseTopicId: 20190
 These are all the *breaking changes* -- that is changes that you absolutely have to worry about if you are updating your app from 1.2.x to 1.3. However, we recommend that you also consider the *recommended* changes listed in the other sections below.
 
  - Ensure that your project has a [`package.json`](https://docs.npmjs.com/files/package.json) file, which will be the foundation for npm package installs.  You can create this by running:
-
-       meteor npm init -y
-
+    ```
+    meteor npm init -y
+    ```
  - Files in a directory named `imports/` will no longer load eagerly. (You should probably rename such a directory as it is the basis of our new [module system](#modules)).
 
  - Files within your app named `*.test[s].*`, `*.app-test[s].*`, `*.spec[s].*` and `*.app-spec[s].*` will no longer load eagerly (you should probably rename such a file if it doesn't contain tests, as it will be eagerly loaded by our new [app testing modes](#testing)).

--- a/content/1.4-migration.md
+++ b/content/1.4-migration.md
@@ -12,9 +12,9 @@ These are all the *breaking changes* &mdash; that is, changes you absolutely hav
 <a name="babel-runtime-required"></a>
 
 The `babel-runtime` npm package is generally required as a dependency since the Meteor `babel-runtime` package no longer attempts to provide custom implementations of Babel helper functions. To install the `babel-runtime` npm, run the following command in any Meteor application:
-
-    meteor npm install --save babel-runtime
-
+```
+meteor npm install --save babel-runtime
+```
 New projects created with `meteor create <app-name>` will automatically have this package added to their `package.json`'s `dependencies`.
 
 <h3 id="binary-packages-require-build-toolchain">Binary Packages require a Build Toolchain</h3>

--- a/content/mobile.md
+++ b/content/mobile.md
@@ -101,9 +101,9 @@ A shortcut is to run `sudo xcodebuild -license accept` from the command line. (Y
 <h4>Enabling Xcode command line tools</h4>
 
 After installing Xcode from the Mac App Store, it is still necesssary to enable those tools in the terminal environment.  This can be accompilshed by running the following from the command prompt:
-
+```
     sudo xcode-select -s /Applications/Xcode.app/Contents/Developer
-
+```
 <h3 id="installing-prerequisites-android">Android</h3>
 
 In order to build and run Android apps, you will need to:

--- a/site/package.json
+++ b/site/package.json
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "private": true,
   "hexo": {
-    "version": "3.7.1"
+    "version": "3.9.0"
   },
   "devDependencies": {
     "chexo": "1.0.7",

